rasa/utils/tensorflow/tf_to_numpy.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any, Dict, Optional
+from tensorflow import Tensor
+
+
+def values_to_numpy(data: Optional[Dict[Any, Any]]) -> Optional[Dict[Any, Any]]:
```

Comment:
please also add tests for this

Reply:
there will be no need for this method after https://github.com/RasaHQ/rasa/pull/7438 is merged

Reply:
so i won't add tests here